enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+          # can be applied if and only if they do not contain changes to osImageURL.
+          policy: "AllowSettings|AllowNone"
+```
+
+The primary user of this strategy is `oc` with tentatively planned enhancements to include verbs 
+like:
+```sh
+$ oc adm update worker-nodes start ...
+$ oc adm update worker-nodes pause ...
+$ oc adm update worker-nodes rollback ...
+```
+
+These verbs can leverage the assisted strategy and `pausedUntil` to allow the manual initiation of worker-nodes
+updates after a control-plane update. 
+
+#### MachineConfigPool Manual Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: Manual
+    config:
+      manual:
+        desiredConfig: <rendered-configuration-name>
+```
+
+The manual strategy requests no automated initiation of updates. New and rebooting
+nodes will only receive the desired configuration. From a metrics perspective, this strategy
+is always paused state.
+
+#### Metrics
+
+`cm_change_pending`
+Labels:
+- kind=ClusterVersion|MachineConfigPool|HostedCluster|NodePool
+- object=<object-name>
+- system=<control-plane|worker-nodes>
+Value: 
+- `0`: no material changes are pending.
+- `1`: changes are pending but being initiated.
+- `2`: changes are pending and blocked based on this resource's change management policy.
+- `3`: changes are pending and blocked based on another resource in the change management hierarchy.
+
+`cm_change_eta`
+Labels:
+- kind=ClusterVersion|MachineConfigPool|HostedCluster|NodePool
+- object=<object-name>
+- system=<control-plane|worker-nodes>
+Value: 
+- `-2`: Error determining the time at which changes can be initiated (e.g. cannot check with ClusterVersion / change management hierarchy).
+- `-1`: Material changes are paused indefinitely OR no permissive window can be found within the next 1000 days (the latter ensures a brute force check of intersecting datetimes with hierarchy RRULEs is a valid method of calculating intersection).
+- `0`: Any pending changes can be initiated now (e.g. change management is disabled or inside machine schedule window).
+- `> 0`: The number seconds remaining until changes can be initiated. 
+
+`cm_strategy_enabled`
+Labels:
+- kind=ClusterVersion|MachineConfigPool|HostedCluster|NodePool
+- object=<object-name>
+- system=<control-plane|worker-nodes>
+- strategy=MaintenanceSchedule|Manual|Assisted
+Value: 
+- `0`: Change management for this resource is not subject to this enabled strategy (**does** consider hierarchy based disable).
+- `1`: Change management for this resource is directly subject to this enabled strategy.
+- `2`: Change management for this resource is indirectly subject to this enabled strategy (i.e. only via control-plane override hierarchy).
+- `3`: Change management for this resource is directly and indirectly subject to this enabled strategy.
+
+#### Change Management Status
+Each resource which exposes a `.spec.changeManagement` stanza should also expose `.status.changeManagement` . 
```

Comment:
```suggestion
Each resource which exposes a `.spec.changeManagement` stanza must also expose `.status.changeManagement` . 
```

I don't think this should be optional.  Given the layers in change management, it's important to get status for any configured change reflected in the status of any resource that is impacted by the status of those other layers.